pkg/logs/internal/launchers/container/tailerfactory/file.go

```diff
@@ -152,8 +162,10 @@ func (tf *factory) findDockerLogPath(containerID string) string {
 				podmanLogsBasePath, "storage/overlay-containers", containerID,
 				"userdata/ctr.log")
 		}
+		dockerLogsPath := dockerLogsBasePathNix
+
 		return filepath.Join(
-			dockerLogsBasePathNix, "containers", containerID,
+			dockerLogsPath, "containers", containerID,
```

Comment:
the `dockerLogsPath` variable doesn't feel useful after your recent refactor. Lets switch back to the prev version where each branch of the `switch` uses either `dockerLogsBasePathWin` or `dockerLogsBasePathNix` directly.